tools/azure-pipelines/debug_files_utils.sh

```diff
@@ -21,7 +21,15 @@ function prepare_debug_files {
 	MODULE=$@
 	export DEBUG_FILES_OUTPUT_DIR="$AGENT_TEMPDIRECTORY/debug_files"
 	export DEBUG_FILES_NAME="$(echo $MODULE | tr -c '[:alnum:]\n\r' '_')-$(date +%s)"
+	# make environment variables available in AzureCI workflow configurations
 	echo "##vso[task.setvariable variable=DEBUG_FILES_OUTPUT_DIR]$DEBUG_FILES_OUTPUT_DIR"
 	echo "##vso[task.setvariable variable=DEBUG_FILES_NAME]$DEBUG_FILES_NAME"
+
+	# make environment variables available in Github Actions workflow configuration
+	if [ ! -z "${GITHUB_OUTPUT+x}" ] && [ -f "$GITHUB_OUTPUT" ]; then
```

Comment:
should we have an explicit azure branch?

Reply:
I see - the [change in the upstream PR](https://github.com/apache/flink/pull/23970#discussion_r1445909482) actually resolves the issue.